src/OpenConext/Profile/Entity/AuthenticatedUser.php

```diff
@@ -34,7 +34,7 @@
     private $nameId;
 
     /**
-     * @var AttributeSet
+     * @var AttributeSetInterface
```

Comment:
Come to think of it, this interface, returned from the factory, is actually a BC break, innit?